<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecentFile</class>
 <widget class="QWidget" name="RecentFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>32</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">RecentFile
{
	background-color: transparent;
}

#lTransferType, #lFileType, #lGetLink
{
	border: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="lTransferType">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources_linux.qrc">
       <normaloff>:/images/download_item_ico.png</normaloff>:/images/download_item_ico.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="lFileType">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources_linux.qrc">
       <normaloff>:/images/small_aftereffects.png</normaloff>:/images/small_aftereffects.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lFileName">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Image.jpeg</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="lGetLink">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources_linux.qrc">
       <normaloff>:/images/clear_item_ico.png</normaloff>:/images/clear_item_ico.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources_linux.qrc"/>
 </resources>
 <connections/>
</ui>
